### PR TITLE
Feature/discover pages content

### PR DESCRIPTION
**Pull request by fishoutoforder**
Merges content updates to the Discover pages for the new website from my forked branch into the main repo's **develop** branch., with the exception of the Technology subpage (which was populated in [a commit merged while this pull request was still open](https://github.com/ceph/ceph.io/commit/370d64e9b384d22f29ccc245fb7515d55d94acbb) - so my updates for that page no longer needed).

- Updates all 'Lorem ipsum' placeholder content with content relevant to the page in question.
- Fixes some minor typos and grammatical fuzziness in the existing Discover page text
- Transfers the content from the Ceph @ home case study to the dev site
- Transfers the existing SoftIron case study content to the dev site
- Adds images for the case studies above

@Pete-Robelou @adamduncan - Just flagging so that you guys can eyeball if necessary, haven't touched any design elements, just the text content itself (plus a few new images in assets/bitmaps)

Signed-off-by: Wendy White <wendy@fishoutoforder.com>

### CODE REVIEW

**Comment by adamduncan** [comment]
Thanks @fishoutoforder. I'll leave for @Pete-Robelou to feed in on any markup/convention particulars.

Just worth pointing out that `/en/discover/technology` has changed and been merged since this PR was opened (https://github.com/ceph/ceph.io/pull/175) — it may be worth excluding that from this particular PR and assessing any tweaks to the latest content if need be

**Comment by fishoutoforder** [comment]
Have gone ahead with @adamduncan's [suggestion](https://github.com/ceph/ceph.io/pull/172#issuecomment-861687810) and removed the commit from this branch that updated the content in the Technology page, given this page now been updated in a separate merge, #175, which occurred after this PR was opened. There are now no conflicts with anything in the develop branch.

**Comment by Pete-Robelou** [comment]
Hi @fishoutoforder.

Apologies for not reviewing this PR earlier but it's the first thing on my list for the morning. I'm sure everything will be fine as it's only text changes.

File: src/en/discover/case-studies/hyperdrive-ceph-in-a-box/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: "HyperDrive: Ceph-in-a-box"
+image: "/assets/bitmaps/ceph-in-a-box.jpg"
```
Comment (Pete-Robelou) [inline src/en/discover/case-studies/hyperdrive-ceph-in-a-box/index.md:3]: Please see my other comment about a naming convention with a prefix.

You'll need to update this if you add the prefix.

File: src/en/discover/case-studies/ceph-for-openstack/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Ceph done right for OpenStack"
+image: "/assets/bitmaps/ceph-in-a-box.jpg"
```
Comment (Pete-Robelou) [inline src/en/discover/case-studies/ceph-for-openstack/index.md:3]: Please see my other comment about a naming convention with a prefix.

You'll need to update this if you add the prefix.

Should this be using `openstack-softiron.png` ?

**Review by Pete-Robelou — COMMENTED** [review]
The file extensions for the image assets are case sensitive and should all be lowercase.

If you could update `photo-alexandre.PNG` to `photo-alexandre.png`.

I've been trying to maintain a consistent naming convention for the assets, prefixing with `logo`, `photo`, `icon`  etc... I appreciate this may not be feasible going forward with multiple content editors and large content updates but it does help with maintenance of the site.

Not essential but I would suggest `photo-ceph-in-a-box.jpg` and `illustration-openstack-softiron.png`. 

I've left other comments where you would need to make updates if you implement the prefix naming convention.

File: src/en/discover/case-studies/ceph-at-home/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Ceph@home: the domestication of a wild cephalopod"
+image: "/assets/bitmaps/alexandre-oliva.png"
+sponsor:
+  name: "Alexandre Oliva"
+  logo: "/assets/favicons/android-chrome-192x192.png"
```
[the remark this answers is not in the document]
Comment (fishoutoforder) [reply src/en/discover/case-studies/ceph-at-home/index.md]: Cheers - I've now added this logo in the assets folder as logo-ceph.png as part of the commits in this branch.

File: src/en/discover/case-studies/ceph-at-home/index.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Ceph@home: the domestication of a wild cephalopod"
+image: "/assets/bitmaps/alexandre-oliva.png"
```
[the remark this answers is not in the document]
Comment (fishoutoforder) [reply src/en/discover/case-studies/ceph-at-home/index.md]: Have now updated the case for this file extension and the file name reference.

File: src/en/discover/case-studies/hyperdrive-ceph-in-a-box/index.md (hunk 1 above)
Comment (fishoutoforder) [reply src/en/discover/case-studies/hyperdrive-ceph-in-a-box/index.md:3]: Have now re-named the image file to use the prefix as recommended.

File: src/en/discover/case-studies/ceph-for-openstack/index.md (hunk 2 above)
Comment (fishoutoforder) [reply src/en/discover/case-studies/ceph-for-openstack/index.md]: Have now updated the image file to use the recommended prefix, and amended the reference in index.md accordingly.

**Comment by fishoutoforder** [comment]
Hi @Pete-Robelou - I've now made the amendments to image naming/refs [as recommended](https://github.com/ceph/ceph.io/pull/172#pullrequestreview-686107036). 

Also @adamduncan I've gone with your suggestion elsewhere to remove the sponsor metadata fields from the Ceph @ Home case study  `\src\en\discover\case-studies\ceph-at-home\index.md`

**Comment by adamduncan** [comment]
> I've gone with your suggestion elsewhere to remove the sponsor metadata fields from the Ceph @ Home case study `\src\en\discover\case-studies\ceph-at-home\index.md`

@Pete-Robelou Some Case studies, such as this, mightn't have sponsors. We should omit the sponsor-related stuff in the right-hand sidebar if that's the case. 

At the moment, there's a bit of a shell that remains when `sponsor` isn't present:

Screenshot 2021-06-18 at 08 49 18

Let's move forward with this PR, but get you to create a separate issue/PR for a solution there

File: src/en/discover/case-studies/ceph-for-openstack/index.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Ceph done right for OpenStack"
+image: "/assets/bitmaps/illustration-openstack-softiron.jpg"
```
Comment (adamduncan) [inline src/en/discover/case-studies/ceph-for-openstack/index.md:3]:
```suggestion
image: "/assets/bitmaps/illustration-openstack-softiron.png"
```

@fishoutoforder Quick one here. Just a correction on the file extension. Should be `png`

File: src/en/discover/case-studies/hyperdrive-ceph-in-a-box/index.md (hunk 1 above)
Comment (adamduncan) [reply src/en/discover/case-studies/hyperdrive-ceph-in-a-box/index.md:3]: @fishoutoforder Thanks. Now the asset filename's been changed, we'll need to update the `image` path here too:

```suggestion
image: "/assets/bitmaps/photo-ceph-in-a-box.jpg"
```

**Review by adamduncan — CHANGES_REQUESTED** [review]

**Review by adamduncan — APPROVED** [review]
Thanks @fishoutoforder. This looks good to go 🤔

**Comment by Pete-Robelou** [comment]
> Let's move forward with this PR, but get you to create a separate issue/PR for a solution there

@adamduncan Agreed, I'll make a ticket for this. There's a similiar issue with blog posts without tags.

**Pull request merged**